### PR TITLE
Implemented setting checkboxes in MainActivity

### LINKED ISSUE #131: Implementing Checkboxes in MainActivity
**Actual Behaviour**
Settings checkboxes used in MainActivity are not being used.

**Expected Behaviour**
The settings should be sent to Program Activities.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by hssahdev**
Fixes #131 

**Changes**: Settings from checkboxes in MainActivity are now passed on to ProgramModeActivity

**Checklist**: 
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by hssahdev** [comment]
@rctay @iamareebjamal Please review!

File: app/src/main/java/io/neurolab/MainActivity.java [hunk 1]
```diff
@@ -68,11 +67,20 @@ protected void onCreate(Bundle savedInstanceState) {
...
     }
 
-    private void startProgramModeActivity(int toastMessageID, int mode){
+    private void startProgramModeActivity(int toastMessageID, int mode) {
+        //Store Settings
+        boolean SETTING_SIMULATION = ((CheckBox) findViewById(R.id.cb_simulation)).isChecked();
+        boolean SETTING_LOAD_RESOURCES_FROM_PHN = ((CheckBox) findViewById(R.id.cb_load_resources_from_phone)).isChecked();
+        boolean SETTING_AUDIO_FEEDBACK = ((CheckBox) findViewById(R.id.cb_audio_feedback)).isChecked();
+        boolean SETTING_24BIT = ((CheckBox) findViewById(R.id.cb_24bit)).isChecked();
+        boolean SETTING_ADVANCED = ((CheckBox) findViewById(R.id.cb_advanced_mode)).isChecked();
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/MainActivity.java]: These are not constants

File: app/src/main/java/io/neurolab/MainActivity.java [hunk 2]
```diff
@@ -68,11 +67,20 @@ protected void onCreate(Bundle savedInstanceState) {
...
+
         Toast.makeText(MainActivity.this, toastMessageID, Toast.LENGTH_SHORT).show();
         Intent intent = new Intent(MainActivity.this, ProgramModeActivity.class);
         Bundle bundle = new Bundle();
         bundle.putInt(ProgramModeActivity.INTENT_KEY_PROGRAM_MODE, mode);
+        boolean[] settings = {SETTING_SIMULATION, SETTING_LOAD_RESOURCES_FROM_PHN, SETTING_AUDIO_FEEDBACK, SETTING_24BIT, SETTING_ADVANCED};
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/MainActivity.java]: Use separate key for each setting

**Comment by hssahdev** [comment]
How about now? @iamareebjamal

File: app/src/main/java/io/neurolab/MainActivity.java [hunk 3]
```diff
@@ -69,18 +69,21 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void startProgramModeActivity(int toastMessageID, int mode) {
         //Store Settings
-        boolean SETTING_SIMULATION = ((CheckBox) findViewById(R.id.cb_simulation)).isChecked();
-        boolean SETTING_LOAD_RESOURCES_FROM_PHN = ((CheckBox) findViewById(R.id.cb_load_resources_from_phone)).isChecked();
-        boolean SETTING_AUDIO_FEEDBACK = ((CheckBox) findViewById(R.id.cb_audio_feedback)).isChecked();
-        boolean SETTING_24BIT = ((CheckBox) findViewById(R.id.cb_24bit)).isChecked();
-        boolean SETTING_ADVANCED = ((CheckBox) findViewById(R.id.cb_advanced_mode)).isChecked();
+        boolean setting_simulation = ((CheckBox) findViewById(R.id.cb_simulation)).isChecked();
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/MainActivity.java]: This isn't python, use camel case

File: app/src/main/java/io/neurolab/MainActivity.java [hunk 4]
```diff
@@ -69,21 +69,21 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void startProgramModeActivity(int toastMessageID, int mode) {
         //Store Settings
-        boolean setting_simulation = ((CheckBox) findViewById(R.id.cb_simulation)).isChecked();
-        boolean setting_load_resources_from_phn = ((CheckBox) findViewById(R.id.cb_load_resources_from_phone)).isChecked();
-        boolean setting_audio_feedback = ((CheckBox) findViewById(R.id.cb_audio_feedback)).isChecked();
-        boolean setting_24bit = ((CheckBox) findViewById(R.id.cb_24bit)).isChecked();
-        boolean setting_advanced = ((CheckBox) findViewById(R.id.cb_advanced_mode)).isChecked();
+        boolean SettingSimulation = ((CheckBox) findViewById(R.id.cb_simulation)).isChecked();
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/MainActivity.java]: Only class names start with capital

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**